### ISSUE TITLE
Can't remove DRM Kindle for PC

### ISSUE THREAD

**Issue by kirstyannepollock** [apprenticeharper/DeDRM_tools#1038]
Kindle for PC 1.24.3
Calilbre 4.13 with Plugins -
  DeDRM 6.6.3
  KFX-ZIP 1.20

All on same machine.

I keep getting a cannot convert due to "locked by DRM" error. 

Not sure DeDRM picked up my key, went in manually to the plugin and hit + and default_key.

Any clues?

Thanks

**Comment by ElleKayEm** [comment]
Post a log:

https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md#i-cannot-solve-my-problem-with-the-dedrm-plugin-and-now-i-need-to-post-a-log-how-do-i-do-that

**Comment by kirstyannepollock** [comment]
Of course. Sorry. Thanks for the link. On 28 Mar 2020 10:02, ElleKayEm <notifications@github.com> wrote:
Post a log:
https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md#i-cannot-solve-my-problem-with-the-dedrm-plugin-and-now-i-need-to-post-a-log-how-do-i-do-that

**Comment by sicklittlemonkey** [comment]
[Method 2](https://www.mobileread.com/forums/showthread.php?t=283371) should work.

**Comment by ElleKayEm** [comment]
And Method 4 with that setup.

**Comment by kirstyannepollock** [comment]
Thanks to all, I'll try 2 and / or 4!On 24 Apr 2020 11:02, ElleKayEm <notifications@github.com> wrote:
And Method 4 with that setup.
